Users/Tutorial.rst

```diff
@@ -264,24 +267,34 @@ That looks horrible, doesn't it? Let's fix it!
     $ coala -S python.bears=PEP8Bear python.files=\*\*/\*.py \
     --apply-patches --save
     # other output ...
+    Executing section cli...
     Executing section python...
     [INFO][11:03:37] Applied 'ApplyPatchAction' for 'PEP8Bear'.
     [INFO][11:03:37] Applied 'ApplyPatchAction' for 'PEP8Bear'.
 
 coala would now fix all spacing issues and without bothering you again.
 
+.. note::
+
+    When you try the above example, you may get a warning, saying that all
+    settings in the ``cli`` section are implicitly inherited to all
+    other sections (if they do not override their values). It also advises
+    us to change the name of that section to avoid unexpected behavior.
+    Read the next section to understand what it means and how you can avoid
+    it.
+
```

Comment:
This line should go something like this:
"The next section explains..."...

Reply:
Okay, yes that's a possible narrative. I'll make the change.